futures/Cargo.toml

```diff
@@ -14,6 +14,7 @@ telemetry = ["iroha_futures_derive/telemetry"]
 [dependencies]
 iroha_futures_derive = { version = "=2.0.0-pre-rc.7", path = "derive" }
 iroha_logger = { version = "=2.0.0-pre-rc.7", path = "../logger" }
+iroha_config = { version = "=2.0.0-pre-rc.7", path = "../config" }
```

Comment:
Increased coupling. I thought we would actually decrease it.

Reply:
Could we move the tests out of futures?

Reply:
Created #2745.